src/install.ts

```diff
@@ -81,9 +81,12 @@ class UI {
     }
   }
 
+  private _pathUpdated: Promise<void>|undefined;
+  private _resolvePathUpdated: (() => void)|undefined;
+
   async promptReload(message: string) {
-    if (await vscode.window.showInformationMessage(message, 'Reload window'))
-      vscode.commands.executeCommand('workbench.action.reloadWindow');
+    await this._pathUpdated
```

Comment:
We should null out `_pathUpdated` after having awaited it, to avoid any objects being unnecessarily kept alive